CRM/Event/Form/ManageEvent/EventInfo.php

```diff
@@ -157,7 +157,7 @@ public function buildQuickForm(): void {
 
     $this->add('textarea', 'summary', ts('Event Summary'), $attributes['summary']);
     $this->add('wysiwyg', 'description', ts('Complete Description'), $attributes['event_description'] + ['preset' => 'civievent']);
-    $this->addElement('checkbox', 'is_public', ts('Display the event in public listings'));
+    $this->addElement('checkbox', 'is_public', ts('Include in Upcoming Events'));
```

Comment:
Today I learned: this setting does not determine whether an event is public, i.e. on whether the `event/info` page is public. It really just determines visibility on Upcoming Events (which is now documented on the help text).

(so I guess this is OK to break translations)

Although sorry to nitpick: wouldn't it be better to say "Display in Upcoming Events"? "include" is not very clear to me. (asking as a non-native-English-speaker)